### PR TITLE
Fix geojson user guide plot

### PR DESCRIPTION
**Pull request by birdsarah**
Specify x and y so plot renders

### CODE REVIEW

File: sphinx/source/docs/user_guide/geo.rst [hunk 1]
```diff
@@ -30,10 +30,11 @@ Bokeh's ``GeoJSONDataSource`` can be used almost seamlessly in place of Bokeh's
     geo_source = GeoJSONDataSource(geojson=geojson)
 
     p = figure()
-    p.circle(alpha=0.9, source=geo_source)
+    p.circle(x='x', y='y', alpha=0.9, source=geo_source)
```
Comment (mattpap) [inline sphinx/source/docs/user_guide/geo.rst]: As to my knowledge how data specs work, this shouldn't be necessary. But maybe this changed recently (changes by Havoc).

Comment (mattpap) [inline sphinx/source/docs/user_guide/geo.rst]: Apparently this doesn't work as I expected anymore:
```
In [1]: from bokeh.plotting import *

In [2]: p = figure()

In [3]: g = p.line()

In [4]: map(type, [g.glyph.x, g.glyph.y])
Out[4]: [NoneType, NoneType]
```
This used to give `['x', 'y']`.

**Comment by birdsarah** [comment]
@mattpap exactly which is why demo was broken

**Comment by birdsarah** [comment]
@havocp @bryevdv if you want to chip in with any context that'd be great. but this definitely is a fix for now.

**Comment by mattpap** [comment]
@birdsarah, this can be safely merged. It was just weird to me seeing something not working that used to work before.

**Comment by birdsarah** [comment]
> @birdsarah, this can be safely merged. It was just weird to me seeing something not working that used to work before.
totally.

**Comment by damianavila** [comment]
LGTM, merging now...

**Pull request merged**